### DISCUSSION TITLE
Discussion on iDNA becoming a flatcoin

### DISCUSSION THREAD

**Discussion by simontakouridis** [idena-network/idena-docs#172] [IIPs]
All successful money in history had the ability hold a stable price. Furthermore, all money in history have failed due to their inability to maintain that stable price. Price stability is imperative because it allows for certainty and confidence in long-term financial contracts. This is something largely missing in crypto, with exception to stablecoins (which are largely not decentralised), so there is a real opportunity to be the first mover.

I think iDNA should become a flatcoin, basically a coin that maintains a flat price (compared with a CPI basket of goods and services). This can be done controlling iDNA supply by reducing or expanding the amount of that all holder addresses have. As an example, if an address had 1 iDNA, it could be expanded (as a %) to 1.25 iDNA, and this would happen for ALL addresses that hold iDNA. 

Ampleforth is a project in the crypto-sphere that does exactly that by leveraging oracles, but it is only a token. To begin with, Idena could do something similar by using quadratic oracles to track the price of iDNA, USD and CPI to adjust the supply of iDNA accordingly.

**Comment by Ed-Relight** [comment]
Imagine believing that cherry-picked CPI has anything to do with the real rate of inflation lmaooo

**Reply by simontakouridis** [reply]
It's designed to be a start, eventually I imagine the oracles would track their own basket of goods/services and not rely on the US official numbers, but that would be some way off due to complexity. Nonetheless, it doesn't need to be perfect to begin with - look at stablecoins for instance (e.g. USDT), they are wildly successful in terms of current utility and they include the inflation component. I believe Idena has the opportunity to capture some of this market as a decentralised inflation adjusted alternative.

**Comment by simontakouridis** [comment]
Better doing this on L2.
